examples/tess.py

```diff
@@ -739,27 +739,8 @@ def check_std(time, flux, dt=0.5): # dt = [hr]
 
         # create AAVSO formatted csv
         csv_lk = OutputFiles(myfit, prior, infoDict, planetdir)
-        try:
-            csv_lk.aavso_csv(airmass,u0,u1,u2,u3,tmidstr)
-            csv_lk.aavso(airmass,u0,u1,u2,u3, tmidstr)
-        except:
-            print(f"Failed to create AAVSO csv for {args.target} - Sector {lcdata['sector']}")
+        csv_lk.aavso_csv(airmass,u0,u1,u2,u3,tmidstr)
+        csv_lk.aavso(airmass,u0,u1,u2,u3, tmidstr)
```

Comment:
any reason to remove this try/except, @pearsonkyle ?

Reply:
It is not necessary and complicates debugging if you can't get a traceback on the error. The errors have been fixed and the code runs without it